Makefile

```diff
@@ -290,7 +290,7 @@ clean:
 # avoid old build problems by removing potentially incorrect old files
 	rm -f config.mak op-i386.h opc-i386.h gen-op-i386.h op-arm.h opc-arm.h gen-op-arm.h
 	rm -f qemu-options.def
-	find . \( -name '*.l[oa]' -o -name '*.so' -o -name '*.dll' -o -name '*.mo' -o -name '*.[oda]' \) -type f -exec rm {} +
+	find . -path ./hw/net/rocker/p4lib -prune -o \( -name '*.l[oa]' -o -name '*.so' -o -name '*.dll' -o -name '*.mo' -o -name '*.[oda]' \) -type f -exec rm {} +
```

Comment:
Make clean at the top level was removing all libraries in all directories.
Since P4libs are currently not built as part of emu build they should not
be removed. These are checked-in libraries until we find a way to compiler
P4 program as part of emu build.
I added the above so that make clean does not remove the P4 libs. Could not
find more elegant way.

On Sun, Apr 19, 2015 at 9:25 PM, Scott Feldman <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663482>:
>
> > @@ -290,7 +290,7 @@ clean:
> >  # avoid old build problems by removing potentially incorrect old files
> >  	rm -f config.mak op-i386.h opc-i386.h gen-op-i386.h op-arm.h opc-arm.h gen-op-arm.h
> >  	rm -f qemu-options.def
> > -	find . \( -name '*.l[oa]' -o -name '*.so' -o -name '*.dll' -o -name '*.mo' -o -name '*.[oda]' \) -type f -exec rm {} +
> > +	find . -path ./hw/net/rocker/p4lib -prune -o \( -name '*.l[oa]' -o -name '*.so' -o -name '*.dll' -o -name '*.mo' -o -name '*.[oda]' \) -type f -exec rm {} +
>
> What does this do? Doesn't seem like we want rocker stuff in the top-level
> Makefile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663482>.
>